### ISSUE TITLE
Can't select category in 0.4

### ISSUE THREAD

**Issue by jaydson** [keystonejs/keystone#1796]
When I click on the category i get this error:
```
values.map is not a function 
```
This is a WIP?

![screenshot from 2015-11-10 22 08 47](https://cloud.githubusercontent.com/assets/72206/11080035/a45fe0e6-87f7-11e5-8531-9eab1e40101f.png)

**Comment by jaydson** [comment]
Perhaps #1793 is related.

**Comment by JedWatson** [comment]
0.4 is definitely still in development, but that shouldn't be happening. You're running this locally against the latest changes in master?

**Comment by JedWatson** [comment]
(that other case is a different issue, btw, in this case something that's expect to be an array isn't, specifically the value of the relationship field)

**Comment by jaydson** [comment]
Yes, I'm running locally pointing to the master branch.

My `package.json`:
```
"keystone": "git+https://github.com/keystonejs/keystone.git"
```
If you give me some tips, like where is this select component, I can try to investigate by myself.

Thanks @JedWatson

**Comment by jossmac** [comment]
Resolved by #1811
